UKHomeOffice/nmsw-ui
# NMSW-1097 Update wording on 'go to next page' button after FAL6 uploaded

# Ticket

NMSW-1097

----

## AC

New wording for button as it will take you to the UPT check page

----

## To test

- Run app
- Sign in
- Create report
- Upload FAl 6
- > On confirmation page should see a button to check passenger UPT status

----

## Notes

- This button is only a wording change, it will still take you to tasklist page as people page is not built yet
